src/modapi/compatibility.h

```diff
@@ -16,6 +16,7 @@ enum
 	MODAPI_COMPATIBILITY_BROADCAST = 0,
 	MODAPI_COMPATIBILITY_SPRITE,
 	MODAPI_NB_COMPATIBILITY,
+	MODAPI_COMPATIBILITY_GUI,
```

Comment:
(Maybe I already said it somewhere) MODAPI_COMPATIBILITY_GUI should be before MODAPI_NB_COMPATIBILITY to allow MODAPI_NB_COMPATIBILITY to effectively count the number of enumeration (MODAPI_NB_COMPATIBILITY is probably useless here, let's keep it for now)